{
  "contest": 142,
  "handle": "Nethermind",
  "address": "0xBc11295936Aa79d594139de1B2e12629414F3BDB",
  "risk": "2",
  "title": "`cancel()` function does not check if the order already was filled at some point.",
  "issueId": 396,
  "issueUrl": "https://github.com/code-423n4/2022-06-putty-findings/issues/396"
}